packages/fast-components-react-msft/src/pivot/pivot.tsx

```diff
@@ -0,0 +1,240 @@
+                        ? this.props.activeId
+                        : get(this.props.items[0], "id", ""),
+            };
+        }
+
+        this.tabsRef = React.createRef();
+    }
+
+    public componentDidMount(): void {
+        this.ltr = this.getLTR();
+        this.setActiveIndicatorOffset();
+        this.prevTabPanelIndex = this.state.tabPanelIndex;
+    }
+
+    public componentDidUpdate(prevProps: PivotProps, prevState: PivotState): void {
+        if (this.ltr !== this.getLTR()) {
+            this.setActiveIndicatorOffset();
+            this.ltr = this.getLTR();
+        }
+
+        if (this.state.activeId !== prevState.activeId) {
+            this.setActiveIndicatorOffset();
+            this.updateTabPanelIndex();
+        }
+    }
+
+    /**
+     * Renders the component
+     */
+    public render(): React.ReactElement<HTMLButtonElement | HTMLAnchorElement> {
+        return (
+            <BaseTabs
+                {...this.unhandledProps()}
+                ref={this.tabsRef}
+                managedClasses={this.generatePivotClassNames()}
+                activeId={this.state.activeId}
+                onUpdate={this.handleTabsUpdate}
+                items={this.props.items}
+                label={this.props.label}
+            >
+                <span
+                    style={{ transform: `translateX(${toPx(this.state.offsetX)})` }}
+                    className={get(
+                        this.props,
+                        "managedClasses.pivot_activeIndicator",
+                        ""
+                    )}
+                />
+            </BaseTabs>
+        );
+    }
+
+    /**
+     * Returns tabs managedclasses with new carousel-specific JSS
+     */
+    protected generatePivotClassNames(): TabsClassNameContract {
+        return {
+            tabs: get(this.props, "managedClasses.pivot", ""),
+            tabs_tabPanels: this.generateTabPanelsClassNames(),
+            tabs_tabList: get(this.props, "managedClasses.pivot_tabList", ""),
+            tabs_tabContent: get(this.props, "managedClasses.pivot_tabContent", ""),
+            tabs_tabPanelContent: get(
+                this.props,
+                "managedClasses.pivot_tabPanelContent",
+                ""
+            ),
+            tab: get(this.props, "managedClasses.pivot_tab", ""),
+            tab__active: get(this.props, "managedClasses.pivot_tab__active", ""),
+            tabPanel: get(this.props, "managedClasses.pivot_tabPanel", ""),
+            tabPanel__hidden: get(
+                this.props,
+                "managedClasses.pivot_tabPanel__hidden",
+                ""
+            ),
+        };
+    }
+
+    private handleTabsUpdate = (activeTabId: string): void => {
+        this.setState({
+            activeId: activeTabId,
+        });
+    };
+
+    private isSelected(element: HTMLElement): boolean {
+        return element.className.includes("active") === true;

```

Comment:
checking that a class-name includes "active" will fail in production builds when these turn into gibberish. Is there an aria attribute you can use to determine selection?